### PR TITLE
Adapted tabs and added right and left panel

### PR DESCRIPTION
**Pull request by ChloeEmmaA**
I added the left (with menu) and right (with features to track) panels and changed the tabs.
:+1:

### CODE REVIEW

**Comment by Olivier0** [comment]
:eyes: (this emoji means that I started to review this PR)

This PR can be tested online at: https://rawgit.com/ChloeEmmaA/open-sympto/Adapt-tabs-calendrier-settings/web/app.html

**Comment by Olivier0** [comment]
There is some garbage text displayed at the bottom of the app if you scroll down. I believe it could fixed by removing these lines: https://github.com/ChloeEmmaA/open-sympto/blob/Adapt-tabs-calendrier-settings/web/app/index.html#L518-L542

**Comment by ChloeEmmaA** [comment]
Yes, right, I don't know how I could have not find it again.... I will remove it for my next commit, but I wait that you finish reviewing before working again on it to avoid conflict.

**Comment by Olivier0** [comment]
You can fix it now. I will not change the code so there is no risk of conflict. I will only post comments on github.

File: web/app/js/open-sympto.js [hunk 1]
```diff
@@ -4,3 +4,92 @@ var view1=myApp.addView('#view-1');
 var view2=myApp.addView('#view-2',{dynamicNavbar:true});
 var view3=myApp.addView('#view-3');
 var view4=myApp.addView('#view-4');
+
+
+//le calendrier
```
Comment (Olivier0) [inline web/app/js/open-sympto.js]: Comments should be written in English

File: web/app/js/open-sympto.js [hunk 2]
```diff
@@ -4,3 +4,92 @@ var view1=myApp.addView('#view-1');
...
+            calendarInline.nextMonth();
+        });
+    },
+    onMonthYearChangeStart: function (p) {
+        $$('.calendar-custom-toolbar .center').text(monthNames[p.currentMonth] +', ' + p.currentYear);
+    }
+});
+
+
+//les messages
```
Comment (Olivier0) [inline web/app/js/open-sympto.js]: English

**Comment by Olivier0** [comment]
I would add at least a setting tab :gear: and an info tab :info:

**Comment by pletelli** [comment]
Could you squash the (useless) commit d6827b38365d04a0cab1d330f333243fc3302e05, to help keeping a clear commit history ?

File: web/app/js/open-sympto.js (hunk 1 above)
Comment (ChloeEmmaA) [reply web/app/js/open-sympto.js]: OK

File: web/app/js/open-sympto.js (hunk 2 above)
Comment (ChloeEmmaA) [reply web/app/js/open-sympto.js]: OK

**Comment by Olivier0** [comment]
I believe we shouldn't use TAB characters to indent the HTML code. Because of the depth of the hierarchy, that makes files difficult to visualize in a text editor (where TABs are often set to 4 or 8 spaces) or even on github where we have to scroll right to see the whole line. So, I would propose to use two spaces instead of one TAB to indent our HTML code.

**Pull request merged**